File: Plugins/Aspose_Pdf_Java_for_Ruby/lib/asposepdfjava/Pages/insertemptypage.rb
module Asposepdfjava
  module InsertEmptyPage
    def initialize()
    	# The path to the documents directory.
        data_dir = File.dirname(File.dirname(File.dirname(File.dirname(__FILE__)))) + '/data/'
                
        # Open the target document
        pdf = Rjb::import('com.aspose.pdf.Document').new(data_dir + 'input1.pdf')

        # insert a empty page in a PDF
        pdf.getPages().insert(1)

        # Save the concatenated output file (the target document)
        pdf.save(data_dir+ "output.pdf")

        puts "Empty page added successfully!"
    end
  end
end

File: Examples/src/main/java/com/aspose/pdf/examples/AsposePdf/Pages/InsertEmptyPage.java
package com.aspose.pdf.examples.AsposePdf.Pages;

import com.aspose.pdf.examples.Utils;

public class InsertEmptyPage
{
    public static void main(String[] args) throws Exception
    {
        // The path to the documents directory.
        String dataDir = Utils.getDataDir(InsertEmptyPage.class);

        //open first document
        com.aspose.pdf.Document pdfDocument1 = new com.aspose.pdf.Document(dataDir + "input.pdf");

        //insert a empty page in a PDF
        pdfDocument1.getPages().insert(1);

        //save concatenated output file
        pdfDocument1.save(dataDir + "output.pdf");

        System.out.println("Empty page added successfully!");
    }
}




